docs/AboutUs.md

```diff
@@ -3,7 +3,7 @@
 Display | Name | Github Profile | Portfolio 
 --------|:----:|:--------------:|:---------:
 ![Tang Zhi You](https://avatars3.githubusercontent.com/u/49060098?s=400&u=b69b125203fd94bf8db9d77b9e3796db73f86cb6&v=4) | Tang Zhi You | [Github](https://github.com/Zhi-You) | [Portfolio](team/zhi-you.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Joseph Abraham | [Github](https://github.com/josephhhhhhhhh) | [Portfolio](team/joseph.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Lim An Qi | [Github](https://github.com/anqi20) | [Portfolio](team/anqi.md)
-![](https://via.placeholder.com/100.png?text=Photo) | Joel Ng | [Github](https://github.com/joelngyx) | [Portfolio](team/joel.md)
-![Kingston Kuan Jun Xiang](https://avatars0.githubusercontent.com/u/35717847?s=460&u=fd4a6977a95813022f82662ada8694105196b9a0&v=4) | Kingston Kuan Jun Xiang | [Github](https://github.com/kstonekuan) | [Portfolio](team/kstonekuan.md)
+![Joseph Abraham](https://avatars2.githubusercontent.com/u/60458098?s=400&u=b9f8cc1b44e9c8e03d8d9fe83d74338113c3a7f8&v=4) | Joseph Abraham | [Github](https://github.com/josephhhhhhhhh) | [Portfolio](docs/team/joseph.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Lim An Qi | [Github](https://github.com/anqi20) | [Portfolio](docs/team/anqi.md)
+![](https://via.placeholder.com/100.png?text=Photo) | Joel Ng | [Github](https://github.com/joelngyx) | [Portfolio](docs/team/joel.md)
+![Kingston Kuan Jun Xiang](https://avatars0.githubusercontent.com/u/35717847?s=460&u=fd4a6977a95813022f82662ada8694105196b9a0&v=4) | Kingston Kuan Jun Xiang | [Github](https://github.com/kstonekuan) | [Portfolio](docs/team/kstonekuan.md)
```

Comment:
Sorry just need to change this back to without the docs and need to change your own file name to be your github username

Reply:
Gotcha